lib/mjsonwp/mjsonwp.js

```diff
@@ -66,6 +66,14 @@ function checkParams (paramSets, jsonObj) {
     if (paramSets.optional) {
       optionalParams = paramSets.optional;
     }
+
+    // If a validation function was provided, test it with jsonObj as args
```

Comment:
might be a good idea to tell us a bit more about how `validate` works, i.e.

```js
// If a function was provided as the 'validate' key, it will here be called with 
// jsonObj as the param. If it returns something falsy, verification will be
// considered to have passed. If it returns something else, that will be the
// argument to an error which is thrown to the user